webapp/src/sagas.js

```diff
@@ -200,7 +211,12 @@ function* handleParcelFetchRequest(action) {
     yield put({ type: types.fetchParcels.success, parcelStates })
   } catch (error) {
     console.warn(error)
-    yield put({ type: types.fetchParcels.failed, error: error.message })
+    yield put({
+      type: types.fetchParcels.failed,
+      error: error.message,
+      parcels: action.parcels,
```

Comment:
Why are we adding `parcels` here?. This is not wrong or anything, just curious.

Reply:
so that we can then try refetching those parcels based on the original request

Reply:
👍